.github/pull_request_template.md

```diff
@@ -1,20 +1,16 @@
 ### Related Issues
+
 - fixes #issue-number
 
 ### Proposed Changes:
+
  <!--- In case of a bug: Describe what caused the issue and how you solved it -->
  <!--- In case of a feature: Describe what did you add and how it works -->
 
 ### How did you test it?
+
 <!-- unit tests, integration tests, manual verification, instructions for manual tests -->
 
 ### Notes for the reviewer
-<!-- E.g. point out section where the reviewer  -->
 
-### Checklist
-- [ ] I have read the [contributors guidelines](https://github.com/deepset-ai/haystack/blob/main/CONTRIBUTING.md) and the [code of conduct](https://github.com/deepset-ai/haystack/blob/main/code_of_conduct.txt)
-- [ ] I have updated the related issue with new insights and changes
-- [ ] I added tests that demonstrate the correct behavior of the change
-- [ ] I've used one of the [conventional commit types](https://www.conventionalcommits.org/en/v1.0.0/) for my PR title: `fix:`, `feat:`, `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`.
```

Comment:
I agree on the principle that the checklist failed miserably its purpose, but can we keep a couple of items around, just for reference? For example
```
### Before you click submit, make sure
- you read the [contributors guidelines](https://github.com/deepset-ai/haystack/blob/main/CONTRIBUTING.md) and the [code of conduct](https://github.com/deepset-ai/haystack/blob/main/code_of_conduct.txt)
- you used one of the [conventional commit types](https://www.conventionalcommits.org/en/v1.0.0/) for my PR title: `fix:`, `feat:`, `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`.
```
I personally use the commit type list a lot 😅

Reply:
Right! I use them too. Changing that. 😅

Reply:
Me too!

Reply:
Can I propose "Adding tests and docs" too? So at least I can point it out to them when they don't do it :sweat_smile: